x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/action/TransportGetInferenceModelAction.java

```diff
@@ -100,8 +100,9 @@ private void getSingleModel(
         }));
     }
 
-    private void getAllModels(ActionListener<GetInferenceModelAction.Response> listener) {
+    private void getAllModels(boolean doNotPersistEndpoints, ActionListener<GetInferenceModelAction.Response> listener) {
         modelRegistry.getAllModels(
+            doNotPersistEndpoints == false,
```

Comment:
Why `== false`?

Reply:
See flip flop above. The tests should have caught this but they needed updating to

Reply:
One benefit of always using the "positive" boolean names is that you never need "flips" like this when passing the variable on.